projects/core/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "./tsconfig.lib.json",
+  "compilerOptions": {
+    "declarationMap": false
+  },
   "angularCompilerOptions": {
-    "enableIvy": false
+    "compilationMode": "partial"
```

Comment:
If we enable partial compilation mode, the library will only be compatible with applications using Angular 9+ with Ivy compilation enabled. See https://angular.io/guide/creating-libraries#transitioning-libraries-to-partial-ivy-format for more info.

I think we might want to revert this to the original `"enableIvy": false` for broader compatibility. If we choose not to revert, we need to update the peer dependencies to Angular >= 9 and add a note to the changelog / breaking changes doc that the new release will not work on projects that are still using View Engine.

Reply:
People that are not using angular 9+ can always use version 11 of this plugin. That's the idea.
This will be published as version 12.

Reply:
I think that someone that doesn't use ivy can use older versions...
I agree about minimal version and release notes.

Reply:
The minimum Angular version should indeed be 9 to follow Angular support: https://angular.io/guide/releases#support-policy-and-schedule

Angular 8 is no longer supported.